main.go

```diff
@@ -68,13 +68,11 @@ func main() {
 	}
 
 	if flags.dry == false {
-		resolve := output.Stdout("Attempting to push new tag to GitHub...")
+		output.Stdout("Attempting to push new tag to GitHub...")
```

Comment:
Wow. I'm dumb. I completely forgot that `--output-tag` suppressed output. That makes total sense.

As for removing the options, I think it makes sense for `--output-tag` to be the default. However, I still like keeping the `--dry-run` (if only for testing purposes). Again, that can be a separate PR.